content/C_UsageGuidelines.md

```diff
@@ -0,0 +1,26 @@
+---
+title : Appendix C Usage guidelines
+---
+
+# Appendix C: Usage guidelines
```

Comment:
We should thinking about codifying these as a Roslyn Analyzer.

Reply:
Well we're always ready to consider PRs. https://github.com/dotnet/reactive/issues/2068